src/App.tsx

```diff
@@ -1,10 +1,13 @@
 import TutorialPage from "./pages/TutorialPage";
 import "./App.css";
+import { TwButton } from "./components/TwButton/TwButton";
+import Nav from "./pages/AfterLogin/Nav";
+import Render from "./pages/AfterLogin/Render";
 
 function App() {
   return (
     <div className="App">
-      <TutorialPage />
+      <Render/>
```

Comment:
For now we can merge this as is, but i'll work on creating a general layout and routing

Reply:
Fixed